queries.js

```diff
@@ -7,26 +7,14 @@ module.exports.GetEnterpriseUserAccount = gql`
         enterprise(slug: $slug) {
             name
             id
-            members(query: $query, first: 1, role: MEMBER) {
```

Comment:
So let me be super clear about the potential concerns here and what the best trade-off workaround.

Concerns:
1. `admin:enterprise` is way too much, no one from Sec would ever agree esp in the cloud SaaS context. It may be fine if it's for an enterprise IT product that really looks into managing the enterprise account.
2. The `query` argument to `members` is just a search string, not the same as `login`. It only takes a public github login name that is a substring match of anyone inside the enterprise to render this enterprise user check inefficient. Therefore, I would not even consider using for some critical check.

Workaround:
1. Do something similar to `GetUser` query instead to check if a user is a member of an enterprise. It seems like an enterprise Oauth app is only able to query the user enterprise information only if the user is a member of that enterprise. Otherwise you would see permission related error, which mean we should use this as another useful input.